tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -3416,7 +3421,7 @@ TEST_F(InputProcessorFixture, getObjectItem_fan_on_off)
 
     EXPECT_EQ(4, NumAlphas);
     EXPECT_TRUE(compare_containers(
-        std::vector<std::string>({"SUPPLY FAN 1", "FANANDCOILAVAILSCHED", "ZONE EXHAUST NODE", "DX COOLING COIL AIR INLET NODE", "", "", ""}),
+        std::vector<std::string>({"SUPPLY FAN 1", "FANANDCOILAVAILSCHED", "ZONE EXHAUST NODE", "DX COOLING COIL AIR INLET NODE", "", "", "General"}),
```

Comment:
Retain case on this last alpha field:

    Fan:OnOff,
      A7 ; \field End-Use Subcategory
           \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
           \type alpha
           \retaincase
           \default General